packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/company-information.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import {
+  H3,
+  FlexContainerResponsive,
+  Content,
+  FlexContainerBasic,
+  GridItem,
+  Grid,
+  Button,
+  Formik,
+  Input,
+  ImageInput,
+  TextArea,
+  SelectBox,
+  Form,
+  LevelRight,
+} from '@reapit/elements'
+import styles from '@/styles/pages/developer-settings-organisation-tab.scss?mod'
+import * as Yup from 'yup'
+import errorMessages from '@/constants/error-messages'
+const { FIELD_REQUIRED } = errorMessages
+
+import { FormFieldInfo } from '@reapit/elements'
+
+export type FieldKeys = 'name' | 'officeEmail'
+export type FormFields = Partial<Record<FieldKeys, string>>
+
+export const formFields: Record<FieldKeys, FormFieldInfo> = {
+  name: {
+    name: 'name',
+  },
+  officeEmail: {
+    name: 'officeEmail',
+  },
+}
+
+const { name, officeEmail } = formFields
+
+const companyInformationFormSchema = Yup.object().shape({
+  [name.name]: Yup.string()
+    .trim()
+    .required(FIELD_REQUIRED),
+  [officeEmail.name]: Yup.string().email(),
+})
+
+export const CompanyInformation = () => {
+  return (
+    <FlexContainerBasic flexColumn hasPadding>
+      <Content>
+        <FlexContainerResponsive flexColumn hasBackground hasPadding>
+          <H3>Company Information</H3>
+          <Formik initialValues={{}} validationSchema={companyInformationFormSchema} onSubmit={() => {}}>
```

Comment:
why onSubmit function return `() => {}` Pls move out and add console.log(values) include //TODO: comment for future easier editable